zxing_standalone/src/main/java/com/google/zxing/client/android/DecodeHandler.java

```diff
@@ -81,9 +83,26 @@ private void decode(byte[] data, int width, int height) {
         rawResult = multiFormatReader.decodeWithState(bitmap);
       } catch (ReaderException re) {
         // continue
-      } finally {
+      } catch (NullPointerException npe) {
```

Comment:
@jarroyoesp Can you please mention in what case(s) we can encounter NPE and AOE?

Reply:
@tarun0 thank you for reply, maybe it's no neccesary so you start the multiFormatReader object in the constructor. You can ignore them, and if you want to read inverted barcodes include the other code. I recommend this PR so in many Samsung devices it's impossible to invert the camera.

Reply:
@jarroyoesp Please remove these two catch blocks and check if it works fine. If it does (i.e. you don't encounter any such exception), and amend the commit. 

Thanks for observing the issue and adding to the work. :)

Reply:
I just removed the uncaught exceptions ;) I use your library and with this code you can scan inverted barcodes/qrs/datamatrix without change noting in the app settings. I tried this with Samsung devices that have  problems inverting the camera (if you try BarcodeScanner it doesn't work in Samsung devices) and it works fine.

Reply:
Thanks for verifying. Lastly, please [squash](http://icoolchar.github.io/git/2017/04/25/git-merge-two-commits-into-one.html) the two commits into one and set the commit message `feat: Added Inverted Barcode scanning feature`. This will assure that one commit does some single purpose. :) 

I'm yet to add guidelines and will add them as soon as I get some time for them.